server/command/debug.go

```diff
@@ -4,131 +4,57 @@
 	if err != nil {
 		return errorOut(params, err)
 	}
 	return out(params, md.JSONBlock(bindings))
 }
 
-func (s *service) executeDebugInstallHTTPHello(params *params) (*model.CommandResponse, error) {
-	params.current = []string{
-		"--app-secret", http_hello.AppSecret,
-		"--url", s.api.Configurator.GetConfig().PluginURL + api.HelloHTTPPath + http_hello.PathManifest,
-		"--force",
-	}
-	return s.executeInstall(params)
-}
-
-func (s *service) executeDebugInstallBuiltinHello(params *params) (*model.CommandResponse, error) {
-	manifest := builtin_hello.Manifest()
-
-	if !s.api.Mattermost.User.HasPermissionTo(params.commandArgs.UserId, model.PERMISSION_MANAGE_SYSTEM) {
-		return errorOut(params, errors.New("forbidden"))
-	}
-
-	app, _, err := s.api.Admin.ProvisionApp(
-		&apps.Context{
-			ActingUserID: params.commandArgs.UserId,
-		},
-		apps.SessionToken(params.commandArgs.Session.Token),
-		&apps.InProvisionApp{
-			Manifest: manifest,
-			Force:    true,
-		},
-	)
-	if err != nil {
-		return errorOut(params, err)
-	}
-
-	conf := s.api.Configurator.GetConfig()
-
-	// Finish the installation when the Dialog is submitted, see
-	// <plugin>/http/dialog/install.go
-	err = s.api.Mattermost.Frontend.OpenInteractiveDialog(
-		dialog.NewInstallAppDialog(manifest, "", conf.PluginURL, params.commandArgs))
-	if err != nil {
-		return errorOut(params, errors.Wrap(err, "couldn't open an interactive dialog"))
-	}
-
-	team, err := s.api.Mattermost.Team.Get(params.commandArgs.TeamId)
-	if err != nil {
+func (s *service) executeDebugAddManifest(params *params) (*model.CommandResponse, error) {
+	manifestURL := ""
+	fs := pflag.NewFlagSet("", pflag.ContinueOnError)
+	fs.StringVar(&manifestURL, "url", "", "manifest URL")
+	if err := fs.Parse(params.current); err != nil {
 		return errorOut(params, err)
 	}
 
-	return &model.CommandResponse{
-		GotoLocation: params.commandArgs.SiteURL + "/" + team.Name + "/messages/@" + app.BotUsername,
-		Text:         fmt.Sprintf("redirected to the DM with @%s to continue installing **%s**", app.BotUsername, manifest.DisplayName),
-		ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
-	}, nil
-}
-
-func (s *service) executeDebugInstallAWSHello(params *params) (*model.CommandResponse, error) {
-	manifest := aws_hello.Manifest()
-
-	if !s.api.Mattermost.User.HasPermissionTo(params.commandArgs.UserId, model.PERMISSION_MANAGE_SYSTEM) {
-		return errorOut(params, errors.New("forbidden"))
-	}
-
-	s.api.Mattermost.Log.Error(fmt.Sprintf("manifest = %v", manifest))
-	app, _, err := s.api.Admin.ProvisionApp(
-		&apps.Context{
-			ActingUserID: params.commandArgs.UserId,
-		},
-		apps.SessionToken(params.commandArgs.Session.Token),
-		&apps.InProvisionApp{
-			Manifest: manifest,
-			Force:    true,
-		},
-	)
-	s.api.Mattermost.Log.Error(fmt.Sprintf("app = %v", app))
-
+	data, err := httputils.GetFromURL(manifestURL)
 	if err != nil {
-		return errorOut(params, err)
+		return nil, err
```

Comment:
Given that `nil` is returned, there is no output to the user. Should we changed that?	i.e. when running `/apps debug-add-manifest --url`